axon-mongo-spring/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I'm likely missing where/how this MockMaker configuration is used. Can you explain this to me, please? :-)

Reply:
To at least have some kind of test for the transaction manager, I mocked a final method. therefore this was needed. I'm not an entirely sure through, I could also just test creation, since really that's all that this library adds.